ntripbrowser/ntripbrowser.py

```diff
@@ -284,13 +284,14 @@ def to_float(arg):
         return [to_float(coordinate) for coordinate in obs_point]
 
     def _get_distance(self, obs_point):
-        if self.coordinates:
-            try:
-                return geodesic(obs_point, self.coordinates).kilometers
-            except ValueError:
-                logger.warning("Unable calculate the geodesic distance between points, %s, %s",
-                               obs_point, self.coordinates)
-        return None
+        if not self.coordinates or None in obs_point:
+            return None
+
+        try:
+            return geodesic(obs_point, self.coordinates).kilometers
+        except ValueError:
+            logger.warning("Unable calculate the geodesic distance between points, %s, %s",
+                           obs_point, self.coordinates)
```

Comment:
```suggestion
        if not self.coordinates or None in obs_point:
            return None

        try:
            return geodesic(obs_point, self.coordinates).kilometers
        except ValueError:
            logger.warning("Unable calculate the geodesic distance between points, %s, %s",
                           obs_point, self.coordinates)
        return None
```

Linter insists that either all branches should return something explicitly, or none of them should.
Also there is a typo in a commit header.